packages/file_selector/file_selector_windows/lib/src/dart_place.dart

```diff
@@ -0,0 +1,12 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Enum for Place
+enum Place {
+  /// TOP
+  top,
+
+  /// BOTTOM
+  bottom
```

Comment:
Nit: consider trailing comma here

```suggestion
  bottom,
```